keptn/keptn
# refactor(installer): Global image and tag management

## This PR treats with #7578

### helm-service + jmeter-service + installer/manifests/keptn
- introduce set values for image registry and repo globally in values.yaml
- introduce common

### installer/manifests/common
- crated `_images.tpl` and implemented functions `service.image` and `service.tag`

### How to test
Example for `installer/manifests/keptn`
- Checkout master 
`helm template . --name-template test-helm-keptn --output-dir ../../../../tmp/keptn-chart-test/tempMaster`
- Checkout PR
`helm template . --name-template test-helm-keptn --output-dir ../../../../tmp/keptn-chart-test/tempPR`
- Compare now templated Manifests
- Test local installation in cluster with custom changes
see `docs/developer/install_local_customisations.md`

Fixes #7578 
Fixes #6043
